### PR TITLE
expose warning() and warningcall() (closes #495)

### LINKED ISSUE #495: Provide Rf_warningcall as R::warning or R::warningcall
See here:  http://thread.gmane.org/gmane.comp.lang.r.packages.devel/852/focus=853

### PR DESCRIPTION
**Pull request by eddelbuettel**
Uses Rf_* versions as usual

### CODE REVIEW

File: inst/include/Rcpp/print.h [hunk 1]
```diff
@@ -26,6 +26,14 @@ inline void print(SEXP s) {
     Rf_PrintValue(s);           // defined in Rinternals.h
 }
 
+inline void warning(const std::string s) {
```
Comment (kevinushey) [inline inst/include/Rcpp/print.h]: `const std::string&`? (missing a `&`)

File: inst/include/Rcpp/print.h [hunk 2]
```diff
@@ -26,6 +26,14 @@ inline void print(SEXP s) {
...
+    Rf_warning(s.c_str());
+}
+
+inline void warningcall(SEXP call, const std::string s) {
```
Comment (kevinushey) [inline inst/include/Rcpp/print.h]: Same here?

Comment (eddelbuettel) [inline inst/include/Rcpp/print.h]: Absolutely. Will fix ASAP.

**Comment by eddelbuettel** [comment]
Oops.  Turns out we had warning() (yet we still redefine it from `Rcpp::Function` in other spots).

**Comment by kevinushey** [comment]
LGTM!

**Comment by eddelbuettel** [comment]
Not my day.  Broke something else, don't quite understand yet what.  Will retreat.

Thank goodness for easily disposable branches :)

**Pull request closed without merging**